lang/pt-BR.json

```diff
@@ -0,0 +1,1010 @@
+        "pdp-non-sale-price-label": "Antes:",
+        "pdp-retail-price-label": "MSRP:",
+        "pdp-custom-fields-tab-label": "Outras informações"
+    },
+    "consent_manager": {
+        "data_collection_warning": "Utilizamos cookies (e outras tecnologias semelhantes) para coletar dados para melhorar a sua experiência de compra.",
+        "accept_all_cookies": "Aceitar todos os cookies",
+        "gdpr_settings": "Configurações",
+        "data_collection_preferences": "Preferências de coleta de dados do site",
+        "use_data_by_cookies": "utiliza dados coletados por cookies e bibliotecas de JavaScript para melhorar a sua experiência de compra.",
+        "data_categories_table": "A tabela abaixo descreve a forma como usamos esses dados por categoria. Para desativar uma categoria de coleta de dados, selecione \"Não\" e salve suas preferências.",
+        "allow": "Permitir",
+        "accept": "aceitar",
+        "deny": "Negar",
+        "dismiss": "Ignorar",
+        "reject_all": "Rejeitar tudo",
+        "category": "Categoria",
+        "purpose": "Propósito",
+        "functional_category": "Funcional",
+        "functional_purpose": "Habilita a funcionalidade aprimorada, como vídeos e bate-papo em tempo real. Se não permitir isso, algumas ou todas essas funções poderão não funcionar corretamente.",
+        "analytics_category": "Análise",
+        "analytics_purpose": "Forneça informações estatísticas sobre o uso do site, como análises da web, para que possamos melhorar o site com o tempo.",
+        "targeting_category": "Direcionamento",
+        "advertising_category": "Publicidade",
+        "advertising_purpose": "Utilizado para criar perfis ou personalizar conteúdo para aprimorar sua experiência de compra.",
+        "essential_category": "Essencial",
+        "esential_purpose": "Essencial para o funcionamento do site e de qualquer serviço solicitado, mas não executa nenhuma função adicional ou secundária.",
+        "yes": "Sim",
+        "no": "Não",
+        "not_available": "N/D",
+        "cancel": "Cancelar",
+        "save": "Salvar",
+        "back_to_preferences": "Voltar às preferências",
+        "close_without_changes": "Você tem alterações que não foram salvas nas suas preferências referentes à coleta de dados. Deseja mesmo fechar sem salvar?",
+        "unsaved_changes": "Algumas alterações não foram salvas",
+        "by_using": "Ao utilizar o nosso site, você concorda com nosso(a)",
+        "agree_on_data_collection": "Ao utilizar o nosso site, você concorda com a coleta de dados conforme descrito em nosso(a) ",
+        "change_preferences": "Você pode alterar as suas preferências quando quiser",
+        "cancel_dialog_title": "Deseja mesmo cancelar?",
+        "privacy_policy": "Política de privacidade",
+        "allow_category_tracking": "Permitir monitoramento de [CATEGORY_NAME]",
+        "disallow_category_tracking": "Não permitir monitoramento de [CATEGORY_NAME]"
+    }
+}
```

Comment:
@BC-tymurbiedukhin should we add a new line at the end of file?

Reply:
updated.